creativecloud/scripts/utils.js

```diff
@@ -86,9 +86,16 @@ function getDecorateAreaFn() {
         import(`${getConfig().codeRoot}/deps/interactive-marquee-changebg/changeBgMarquee.js`);
         break;
       }
-      case firstBlock?.classList.contains('marquee'):
-        firstBlock.querySelectorAll('img').forEach(eagerLoad);
+      case firstBlock?.classList.contains('marquee'): {
+        // Load image eagerly for specific breakpoint
+        const viewport = defineDeviceByScreenSize();
+        if (viewport === 'MOBILE') eagerLoad(firstBlock.querySelector('div:first-child img'));
+        else if (viewport === 'TABLET') eagerLoad(firstBlock.querySelector('div:nth-child(2) img'));
+        else if (viewport === 'DESKTOP') eagerLoad(firstBlock.querySelector('div:last-child img'));
+        // Last image of last column of last row
+        eagerLoad(firstBlock.querySelector('div:last-child > div:last-child img'));
```

Comment:
Not sure if you need that last eagerLoad, if you do, no worries about me removing in suggestion below. I think you could make this a bit cleaner with a data structure, and considering `defineDeviceByScreenSize()` defaults to return 'TABLET', this ought to be safe to use. 

```suggestion
    const lcpImageByViewport = {
        'MOBILE': 'div:first-child img',
        'TABLET': 'div:nth-child(2) img',
        'DESKTOP': 'div:last-child img',
    }
  
    eagerLoad(lcpImageByViewport(viewport));
```

Reply:
Done